### DISCUSSION TITLE
Training of the factorized model

### DISCUSSION THREAD

**Discussion by ildmxGL** [InterDigitalInc/CompressAI#159] [Q&A]
Hi, @fracape 

I trained factorized model for 100 epochs, but inference results for kodak dataset were not similar. 

![factorized small](https://user-images.githubusercontent.com/46182205/189040239-ac76330b-cbfb-4cb3-806e-572d2cc9f69a.jpg) ![factorized heavy](https://user-images.githubusercontent.com/46182205/189040248-9fecbd4b-0804-481c-be7a-a99f53081915.jpg)

"kodak-{number}" means just a model number. The former is the 3.0M model, and the latter is the 7.0M model. The lambda is 0.01; aims quality 3.

Are these vary results usual? Do I have to train several models for best results as the pre-trained?
Or did the models just not converge?

Thanks for your great works.

**Comment by fracape** [comment]
Hi, sorry for the late response, I missed this. If still relevant, which training set did you use? Did you monitor your losses? Your bitrates are very weird.
